native-script.ps1

```diff
@@ -107,12 +105,26 @@ if (!$env:ANDROID_HOME) {
 	refreshenv
 }
 
+# Install OpenJDK after Android SDK, as currently installing the Android SDK also installs the Oracle Java 1.8 version.
+Install "Java Development Kit (OpenJDK)" "Installing Java Development Kit (OpenJDK)" "choco upgrade openjdk --version 11.0 --force --yes"
+
+# For some reason refreshing the environment variables is not working with Chocolatey.
+# To force it, refresh the $PROFILE, as shown here: https://stackoverflow.com/a/46760714/3357767
+$env:ChocolateyInstall = Convert-Path "$((Get-Command choco).path)\..\.."
+Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
+refreshenv
+
 if (!$env:JAVA_HOME) {
-	$curVer = (Get-ItemProperty "HKLM:\SOFTWARE\JavaSoft\Java Development Kit").CurrentVersion
-	$javaHome = (Get-ItemProperty "HKLM:\Software\JavaSoft\Java Development Kit\$curVer").JavaHome
-	[Environment]::SetEnvironmentVariable("JAVA_HOME", $javaHome, "User")
-	$env:JAVA_HOME = $javaHome;
-	refreshenv
+	$openJdkLocation = "C:\Program Files\OpenJDK";
```

Comment:
Is this location fine for both x86 and 64 bit os?

Reply:
No idea :smile:  I'll check it, thanks

Reply:
I've checked it - the path is correct.